### PR TITLE
fix(FEC-11509): fontSize is not working

### PR DESCRIPTION
**Pull request by lianbenjamin**
### Description of the Changes

**issue:**
fontSize configuration is not in use.

**solution:**
removing fontSize configuration.

Solves FEC-11509

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Review by yairans — APPROVED** [review]

**Comment by SivanA-Kaltura** [comment]
@lianbenjamin I also came across this issue because of a ticket I'm working on, FEC-11510. Please note that fontSize is used in https://github.com/kaltura/kaltura-player-js/blob/master/src/common/cast/text-style-convertor.js. We probably can remove this use and replace it with fontScale, but we need to check cast receiver and sender first.

**Comment by SivanA-Kaltura** [comment]
@lianbenjamin fontSize is also used in https://github.com/kaltura/kaltura-player-js/blob/master/test/src/setup.spec.js

File: flow-typed/types/text-style.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 // @flow
 declare type PKTextStyleObject = {
-  fontSize: string,
```
Comment (SivanA-Kaltura) [inline flow-typed/types/text-style.js:3]: Please see my comments about uses in kaltura-player-js

File: src/track/text-style.js [hunk 2]
```diff
@@ -157,7 +156,6 @@ class TextStyle {
   static toJson(text: TextStyle): PKTextStyleObject {
     return {
       fontEdge: text.fontEdge,
-      fontSize: text.fontSize,
       fontScale: text.fontScale,
```
Comment (SivanA-Kaltura) [inline src/track/text-style.js:159]: fontSize is referenced in api.md, so after fontSize is removed the docs should be rebuilt by running yarn run docs:generate

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/track/text-style.js [hunk 3]
```diff
@@ -168,11 +166,9 @@ class TextStyle {
   }
 
   /**
-   * Font size, such as 1, 2, 3...
+   * Font scale, such as 1, 2, 3...
    * @type {number}
    */
-  fontSize: string = '100%';
-
   fontScale: number = 1;
```
Comment (SivanA-Kaltura) [inline src/track/text-style.js:172]: since the jsdoc has been modified, please rebuild api.md by running "yarn run docs:generate"

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

**Comment by OrenMe** [comment]
@SivanA-Kaltura , @lianbenjamin as we decided to keep fontSize and fix the implementation then closing this one

**Pull request closed without merging**